src/osd/modules/debugger/win/consolewininfo.cpp

```diff
@@ -317,9 +317,9 @@ bool consolewin_info::handle_command(WPARAM wparam, LPARAM lparam)
 }
 
 
-void consolewin_info::process_string(char const *string)
+void consolewin_info::process_string(const std::string &string)
```

Comment:
All the code in this module consistently places the `const` to the right of the type.